components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -327,7 +328,7 @@ function CustomLeague:_getGameModes(args, makeLink)
 	end
 
 	local gameModes = mw.text.split(args.gamemode, ',', true)
-	table.foreach(gameModes,
+	Table.foreach(gameModes,
```

Comment:
I'm not sure I understand this one. Is `table` a MW keyword?

Reply:
`table.foreach` is a legacy Lua function, deprecated in 5.1